starter-project-web-react/cypress/e2e/blogPost.spec.cy.ts

```diff
@@ -0,0 +1,63 @@
+describe('Blog Post', () => {
+    beforeEach(()=>{  
+      cy.login(Cypress.env('TEST_EMAIL'), Cypress.env('TEST_PASSWORD')) 
+    })
+  
+    it('should post blog ', () => {
+      cy.visit('/blogs')
+
+      const title = "TEST_TITLE new 1"
+      const content = "Test_Content 1"
+      cy.contains('Post Blog').click()
+      cy.get('[name="title"]').clear()
```

Comment:
is using the "name" as a selector the recommended way ?
I am not sure but I feel like I saw the use of adding a new selector attribute or a class or sth
maybe it is worth checking out because name is a common attribute that may be reused